docs/experiment/experiment_creation.md

```diff
@@ -0,0 +1,75 @@
+One or more of these variables can be added in the experiment summary screen by selecting the experimental variables tab and pressing the "add variables" button within the disclaimer.
+![experiment_summary](images/experimental_summary_no_variables.png)
+
+!!! info "Project role"
+    Should you not see the add button,
+    ensure that you have been granted the "write" or "admin" project role by the project owner/admin!
+
+Once clicked the add variables dialog will appear, in which one or more variables can be defined and linked to your experiment.
+![add_experimental_variables](images/add_experimental_variables.png)
+
+Defined variables can also be deleted easily via a click on the cross icon next to the variable
+![add_experimental_variables_deleted](images/add_experimental_variables_deleted.png)
+
+Once all variables have been defined, they can be linked to the experiment by clicking the "Add" button within the dialog.
+![experiment_summary](images/add_experimental_variables_summary.png)
+
+Finally, already defined experimental variables can be edited by clicking on the edit button within the Experimental Variables tab, 
+which will open the edit experimental variable dialog in which the necessary changes can be made and saved via a click on the save button. 
+![edit_experimental_variables](images/edit_experimental_variables.png)
+
+## Experimental Group Creation
+
+Experimental groups contain the biological replicates which experienced the same condition defined by its experimental variables within your experiment.  
+You can define your samples into distinct groups via a selection of the experimental groups tab within the experiment summary screen. 
+In this tab press the "Add groups" button within the disclaimer.
+![experiment_summary](images/experimental_summary_no_groups.png)
+
+!!! info "Project role"
+    Should you not see the add button,
+    ensure that you have been granted the "write" or "admin" project role by the project owner/admin!
+
+Once clicked the add experimental group dialog will appear, in which one or more groups can be defined via a selection of the previously defined experimental variable and linked to your experiment.
+![add_experimental_variable](images/add_experimental_groups.png)
+
+Defined groups can also be deleted easily via a click on the cross icon next to the group
+![add_experimental_variables_deleted](images/add_experimental_groups_deleted.png)
+
+Once all variables have been defined, they can be linked to the experiment by clicking the "Add" button within the dialog.
+![experiment_summary](images/add_experimental_groups_summary.png)
+
+Finally, already defined experimental groups can be edited by clicking on the edit button within the Experimental Groups tab,
+which will open the edit experimental groups dialog in which the necessary changes can be made and saved via a click on the save button.
+![edit_experimental_groups](images/edit_experimental_groups.png)
```

Comment:
I would add here, that editing is no longer possible (I think?) once samples have been registered. Maybe in one of those info boxes?

Reply:
Good point 👍